### PR TITLE
[docker] Fix docker credential management

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

### Description of PR

Summary: Fix the credential management for docker methods.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Before we hard-coded access to the docker repo credentials which was really insecure and clunky. This PR takes advantage of the creds fixture that has been added to read the necessary credentials from the docker_registry file.

#### How did you do it?
Moved from `secret_vars` to `creds`.

#### How did you verify/test it?
Ran locally, verified that the correct variables are retrieved when `load_docker_registry_info` is called.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -333,6 +333,7 @@ def creds(duthost):
     groups.append("fanout")
     logger.info("dut {} belongs to groups {}".format(duthost.hostname, groups))
     files = glob.glob("../ansible/group_vars/all/*.yml")
+    files += glob.glob("../ansible/vars/*.yml")
     for group in groups:
```
Comment (neethajohn) [inline tests/conftest.py:337]: Is 'all' part of groups list?

Comment (daall) [reply tests/conftest.py:337]: It is, but this file isn't under 'all'.

**Comment by daall** [comment]
retest vsimage please

File: tests/conftest.py (hunk 1 above)
Comment (daall) [reply tests/conftest.py:337]: Also this is `vars` not `group_vars`, that caught me off guard too. :)

**Pull request merged**